src/common/syncjournaldb.cpp

```diff
@@ -1013,6 +1013,13 @@ qint64 SyncJournalDb::keyValueStoreGetInt(const QString &key, qint64 defaultValu
     return _getKeyValueStoreQuery.int64Value(0);
 }
 
+void SyncJournalDb::keyValueStoreDelete(const QString &key)
+{
+    ASSERT(_deleteKeyValueStoreQuery.initOrReset("DELETE FROM key_value_store WHERE key=?1;", _db));
```

Comment:
@mgallien Are you sure? It is said otherwise in the `asserts.h`.

`// Prints a message on failure, even in release builds.` _src/common/asserts.h_

Reply:
I think also this should work in release builds. What an interesting piece of code (the ASSERT code) :)

Reply:
> @mgallien Are you sure? It is said otherwise in the `asserts.h`.
> 
> `// Prints a message on failure, even in release builds.` _src/common/asserts.h_

I see
I already see that it is already like that in other places within this file

I cannot say that I like that or find it easy to read quite the opposite in fact

do you find this code easy to read and understand ?

Can you use it like a normal assert (executing at the line before and using the assert only to check result ?

I really would prefer to get read of all our custom macro rather than relying on them more

Reply:
@mgallien Fixed.